managedtenants/bundles/bundle_builder.py

```diff
@@ -4,6 +4,7 @@
 from pathlib import Path
 
 import semver
+from packaging.version import parse as ParseVersion
```

Comment:
We already require `semver` so easier to use that. (See [here](https://github.com/mt-sre/managed-tenants-cli/blob/e29e76c215e5c63597f65b54d74220fc3a654287/setup.py#L61)).

You can also just use `max` with the `key` [keyword argument](https://python-semver.readthedocs.io/en/latest/usage/get-min-and-max-of-multiple-versions.html).

Reply:
Ohh cool, thanks! Will do so :)